<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>901</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>录屏分帧</string>
    </attribute>
    <widget class="QPushButton" name="lp_start">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>录制视频</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_max_time">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>39</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>录制最大时长</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分帧视频目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_video_file_path">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="lp_choice_video">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>手动选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分帧图片</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="lp_pictures">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>771</width>
       <height>401</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>769</width>
        <height>399</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>FPS测试</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>启动测试</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入启动activity</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_input_activity">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_get_current_activity">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>获取当前activity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_get_start_activity">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>获取启动activity</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="page2_is_first">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>首次启动</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="page2_second">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>二次启动</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>测试次数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_test_num">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_time_test">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始测试</string>
     </property>
    </widget>
    <widget class="QLabel" name="page2_show_result">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>511</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>551</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>关于</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>                                                仿写xx测试工具</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>应用进程</string>
   </property>
  </widget>
  <widget class="QLabel" name="device_id">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设备ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="device_app_list">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget1</sender>
   <signal>currentChanged(int)</signal>
   <receiver>tabWidget1</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
